src/ucp/core/ucp_mm.h

```diff
@@ -121,8 +121,9 @@ typedef struct {
  * Memory type pack/unpack registration context
  */
 typedef struct {
-    ucp_mem_h         ucp_memh;    /* memh from rcache if MD id cacheable */
-    uct_mem_h         uct_memh;    /* memh for specific MD */
+    ucp_md_index_t    md_index; /* index of MD */
```

Comment:
cosmetic: consider aligning the comments

Reply:
git-clang-format complaining on it